sdk/eventhub/mock-hub/src/server/mockServer.ts

```diff
@@ -148,6 +148,8 @@ export class MockServer extends EventEmitter {
     return new Promise((resolve, reject) => {
       const options = this._options;
       const ONE_MB = 1024 * 1024;
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
       const listenOptions: ListenOptions & ConnectionOptions & any = {
```

Comment:
huh, this is a very weird type, I think it will collapse to `any` anyway so not sure what is the point.

Reply:
Yes, agreed, I thought it was a bit odd when I was working with this. The shape is more or less `ListenOptions & ConnectionOptions` with one or two differences, but it's difficult to construct a type that describes it. I think the `ListenOptions & ConnectionOptions` part does provide a bit of info about the type though, so guessing that's why it's there (even if it doesn't provide any validation)?